libs/blocks/aside/aside.js

```diff
@@ -46,32 +51,115 @@ function decorateStaticLinks(el) {
   textLinks.forEach((link) => { link.classList.add('static'); });
 }
 
+function decorateModalImage(el, lnkImg) {
+  if (el.classList.contains('split')) lnkImg.closest('div').classList.add('has-modal');
+  lnkImg.classList.add('play-btn');
+  const playIcon = createTag('div', { class: 'play-icon-container', 'aria-label': 'play' }, PLAY_ICON);
+  const playCircle = createTag('div', { class: 'play-btn-circle', 'aria-label': 'play' }, playIcon);
+  const playContainer = createTag('div', { class: 'play-container', 'aria-label': 'play' });
+  lnkImg.parentNode.appendChild(lnkImg.querySelector('picture'));
+  lnkImg.parentNode.appendChild(playContainer);
+  lnkImg.appendChild(playCircle);
+  playContainer.appendChild(lnkImg);
+}
+
+function decorateMedia(el) {
+  if (!(el.classList.contains('medium') || el.classList.contains('large'))) return;
+  const allMedia = el.querySelectorAll('div > p video, div > p picture');
+  let processed = false;
+  allMedia.forEach((media) => {
+    if (processed) return;
+    const parentP = media.closest('p');
+    const siblingP = parentP?.nextElementSibling;
+    if (!siblingP || siblingP.nodeName.toLowerCase() !== 'p') return;
+    const siblingText = siblingP.textContent;
+    const hasFormats = siblingText.match(FORMAT_REGEX)?.index;
+    if (!(hasFormats === 0)) return;
+    processed = true;
+    const formats = siblingText.split(': ')[1]?.split(/\s+/);
+    const formatClasses = formats ? ['format',
+      `desktop-${formats[((formats.length - 1) > 0) ? (formats.length - 1) : 0]}`,
+      `tablet-${formats[((formats.length - 2) > 0) ? (formats.length - 2) : 0]}`,
+      `mobile-${formats[0]}`,
+    ] : [];
```

Comment:
If you only author one format option `format: wide` you really only need to output that one class `mobile-wide` and not all three breakpoint classes. The css cascade for `mobile-wide` will work across all breakpoints since there is no override at tablet or desktop.

Reply:
Hi @rgclayton 
The format string addition was added to apply the styles as per format `Mobile Tablet Desktop` or `Mobile+Tablet Desktop` or `Mobile+Tablet+Desktop`

I have updated the code to keep it in sync with the review comment and would now apply the style as per `Mobile Tablet Desktop` or `Mobile Tablet+Desktop` or `Mobile+Tablet+Desktop`

Also the styles would now cascade and tablet and desktop classes would be added explicitly when specified.